tests/queries/0_stateless/03533_query_log_table_database_name_consistency_bug.sh

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest, no-parallel
```

Comment:
We should avoid using these tags unless absolutely necessary. Any reason why these are necessary? We should be able to remove them.

Reply:
You won't need to create a database here. You can directly work with the tables and if you want to access the current database, you can use the variable `$CLICKHOUSE_DATABASE`, for example: https://github.com/ClickHouse/ClickHouse/blob/61d8845c3820cf3b7fd249136d7a6e10f0daa3d2/tests/queries/0_stateless/02444_async_broken_outdated_part_loading.sh#L19 There are more examples if you grep around in `.sh` test files. By doing it this way, you will be able to remove the tags as the database name would be unique between tests runs and when run in parallel.

Reply:
Yep, I get it, but what if I need test the database name too, each test, as I can understand, create the `test_{hash}` database but I need to test database name with special characters in its name (f.e `-`)

Reply:
> I added these tags after getting errors while running this workflow (a lot of database already exists errors).

Since we are using static database name and the test needs it, we can avoid running the tests in parallel. But we don't need `no-fasttest` - so I've removed it.